internal/controller/accesskey/accesskey.go

```diff
@@ -241,3 +238,16 @@ func (c *external) Delete(ctx context.Context, mg resource.Managed) (managed.Ext
 func (c *external) Disconnect(ctx context.Context) error {
 	return nil
 }
+
+func connectionDetails(creds *cloudian.SecurityInfo) managed.ConnectionDetails {
+	return managed.ConnectionDetails{
+		"secretKey": []byte(creds.SecretKey),
+		"config.toml": []byte(fmt.Sprintf(
+			`[default]
```

Comment:
Why `[default]` here?

Reply:
It's the default config, see https://marketplace.upbound.io/providers/upbound/provider-family-aws/v1.19.0

![image](https://github.com/user-attachments/assets/10c8b2e4-5df4-4177-83a8-98d103479d92)

Reply:
Thanks!